corehq/apps/cloudcare/urls.py

```diff
@@ -34,8 +34,7 @@
 api_urls = [
     url(r'^login_as/users/$', LoginAsUsers.as_view(), name=LoginAsUsers.urlname),
     url(r'^cases/$', get_cases, name='cloudcare_get_cases'),
-    url(r'^fixtures/(?P<user_id>[\w-]*)/$', get_fixtures, name='cloudcare_get_fixtures'),
```

Comment:
This route is no longer used, and it is not present access logs on the proxy (oldest logs are from July 5th).